src/pages/index.js

```diff
@@ -169,11 +328,18 @@ cardModalCloseBtn.addEventListener("click", () => {
 
 cardFormModalNewPost.addEventListener("submit", handleNewPostFormSubmit);
 
```

Comment:
* Validation of the `avatar` form doesn't work. See the errors when I type anything in the input https://snipboard.io/MtPDw3.jpg